File: python/tests/test_survey_submit.py
#!/usr/bin/env python

import sys
sys.path += ['../']

from epidb.client import EpiDBClient

key = '0123456789abcdef0123456789abcdef01234567'
data = 'data'

client = EpiDBClient(key)
res = client.survey_submit(data)

print res

File: php/tests/test_survey_submit.php
<?

require_once('../EpiDBClient.php');
require_once('config.php');

$key = $_key;
$data = 'data';

$client = new EpiDBClient($key);
$client->server = $_server;
$res = $client->survey_submit($data);

print "$res\n";

